pipes/config.yaml

```diff
@@ -140,6 +140,13 @@ samples_per_run: "samples-runs.txt"
 # to insufficient input data, consider expanding the list of accessions given to lastal.
 accessions_file_for_lastal_db_build: ""
 
+# File containing the list of accessions for reference genomes to use for reference selection.
+# Defaults to the same list as `accessions_for_ref_genome_build`.
+# For multi-segment genomes, the file must list all segments for the first reference, then all segments
+# for the second reference, and so on; for each reference, segments must be listed in the same order as for
+# `accessions_for_ref_genome_build`; all segments must be listed for each reference.
+accessions_file_for_refsel_db_build: ""
```

Comment:
Should this be a file? config["accessions_for_ref_genome_build"] seems to not be so?

Reply:
Or perhaps `accessions_for_ref_genome_build` should be a file...

Reply:
If it's non-blank then it must be a file, but if it's blank then it defaults to a file containing the list of accessions specified in accessions_for_ref_genome_build , as for lastal.